src/adapter/activateDaffodilDebug.ts

```diff
@@ -19,11 +19,15 @@ import { getDebugger, getDataFileFromFolder } from '../daffodilDebugger'
 import { FileAccessor } from './daffodilRuntime'
 import * as fs from 'fs'
 import * as infoset from '../infoset'
-import { getConfig, setCurrentConfig } from '../utils'
+import { getConfig, getCurrentConfg, setCurrentConfig } from '../utils'
```

Comment:
"Confg".... simply a typo that will waste lots of people's time under maintenance. Fix please.